apps/ledger-live-mobile/src/screens/RequestAccount/02-SelectAccount.tsx

```diff
@@ -173,7 +183,7 @@ function SelectAccount({ navigation, route }: Props) {
       ]}
     >
       <TrackScreen category="RequestAccount" name="SelectAccount" />
-      <KeyboardView>
+      <KeyboardView style={styles.root}>
```

Comment:
This screen is common across the different applications. Be sure Diego is aware of this change.